### PR TITLE
NEWS: S3 methods

### PR DESCRIPTION
**Pull request by vincentarelbundock**
Single line commit to add NEWS item for PR #107

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # polars (development version)
 
+## What's changed
+
+* Base R functions work on DataFrame and LazyFrame objects via S3 methods: as.data.frame, as.matrix, dim, head, length, max, mean, median, min, na.omit, names, sum, tail, unique, ncol, nrow (#104 @vincentarelbundock).
```
Comment (eitsupi) [inline NEWS.md:5]: Maybe typo? (104 -> 107)

Because #107 is included in 0.5.0, could you move this item to 0.5.0?

**Review by eitsupi — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 1 above)
Comment (vincentarelbundock) [reply NEWS.md:5]: Is it included? AFAICT from the commit history, the bump occurred before the merge: https://github.com/pola-rs/r-polars/commits/main

Comment (eitsupi) [reply NEWS.md]: @sorhawell was tagged after merging #107.
https://github.com/pola-rs/r-polars/commit/e9d98de423f69a0f94c7b46504741cdc587b389c is tagged as 0.5.0

Comment (vincentarelbundock) [reply NEWS.md]: Ah yes, I see now. Thanks for catching this and clearing things up!

**Review by eitsupi — APPROVED** [review]
Thanks!

**Pull request merged**